### DISCUSSION TITLE
Who's hiring (April 2023)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#47869] [Show and tell]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #47868.

*If you're looking for last month's discussion, see #46889. Several companies [hired someone](https://twitter.com/nutlope/status/1610667757565083648) as a result of it!*

**Comment by zachselby** [comment]
Hydrow, Inc. &nbsp;|&nbsp; E-commerce Principal Engineer &nbsp;|&nbsp; Boston, MA or Remote &nbsp;|&nbsp; Full-time &nbsp;|&nbsp; [Apply Today](https://jobs.lever.co/Hydrow/aa909810-ad4f-4e6b-ae0f-8bed9a822127)

#### Who We Are
[Hydrow](https://hydrow.com) is the Live Outdoor Reality (LOR)™ rowing machine that delivers on-water experiences at home with workouts led by world-class athletes. If you ask us, we’re the perfect mix of athletes, entrepreneurs, engineers, and creatives. Our mission is not just to create an amazing fitness experience, but a beautiful human experience - one that’s honest, inclusive, and positive. At Hydrow, you’ll have the chance to shape a company that is disrupting the fitness industry.

#### Why We Need You
As an E-commerce Principal Engineer working on the Digital Platforms team, you will lead the engineering behind Hydrow’s direct to consumer web channel on a modern, composable stack. You’ll contribute technical leadership and hands-on execution, establish close partnerships with stakeholders including engineering and marketing leadership and guide ongoing feature development for the web. You will be responsible for driving technical excellence and maintaining modern development practices for internal and partner delivery teams working on our headless digital experience stack (Next.js, Contentful, Shopify).

##### You’re excited about this opportunity because you will…
- Help build an all-new in-house e-commerce development team
- Oversee and drive e-commerce architectural direction for a leading consumer brand
- Work with leading edge e-commerce web technologies (including headless and serverless architecture, edge computing) to design and build beautifully immersive, lightning-fast e-commerce experiences

**Comment by jared-leddy** [comment]
Veteran Database | Front-End Web Dev | Remote, USA | Part-time | Volunteer | [VeteranDB](https://veterandb.com)

Front-End:
Node.js
TypeScript
React
Next.js

Familiarity with Nest.js is useful, but not required. 

USA Citizens Only
Veterans get priority

Email resume to [hello@veterandb.com](mailto:hello@veterandb.com).

**Comment by bigsee** [comment]
Påfyllsystemer AS  |  Full Stack Engineer  |  Remote |  Full-time (Contract to Permanent)  | €20-35ph (€40-70k pa) | [Apply now on Upwork](https://www.upwork.com/freelance-jobs/apply/Full-stack-engineer-who-cares-about-people-and-the-environment_~01a094932c82804d96/)

## **About us**

[På(fyll)](https://pafyll.com/en) is a new startup from [Orkla](https://www.orkla.com/about-us/orkla-care/ohpcnorge/), Æra and Bakken & Bæck. We're on a mission to make everyday living easier while tackling a massive problem: the growing amount of plastic waste from household products. We are now looking to grow our purpose-driven, inclusive and flexible team. 

Specifically, we are looking for skilled and emotionally-intelligent engineers to join our cross-functional product team and deliver solutions for the needs of our customers.

## **About the role**

Our engineers will play a key hands-on role in the growth and evolution of På(fyll)’s products based on user insight. You will be responsible for the delivery of solutions to meet user needs and you will deliver regular, incremental value. This will include maintenance and development of our existing user-facing products as well as building new internal ones for analysis of our operations and customer experience, which will provide the crucial foundation for our expansion plans.

You can read more about the team and project [here](https://pafyll.notion.site/Software-Engineer-c8daefd2de044cb1b2b2668beaaa6246). If you would like more information, please email carlton@pafyll.com. If you would like to apply to us directly, please include your CV, a link to your Github and details of your availability and salary expectations.

#### Individuals only please. Please do not contact me if you represent an agency.

**Comment by BartoszWithlocals** [comment]
Withlocals | Senior Software Engineer | Eindhoven, Netherlands or Remote | Full-time | [www.withlocals.com](https://www.withlocals.com/)

### About Withlocals
Withlocals envisions a world where responsible travel experiences are universally accessible and tourism positively impacts our planet, changing lives for the better.

We’re on a mission to reshape the way we travel. We aim to connect people and cultures through unique and memorable travel experiences with people, places, and the planet in mind. To travel for the better, for the experience, and for the sense of community.

“Connecting people and cultures is the very essence of humanity. It is through these connections that we can learn from one another, grow together, and build a better world for all.”

We are a fast-growing and dynamic company with a diverse and passionate team of 22 people based in our Eindhoven office and working remotely from different countries. Our tech team consists of 11 brilliant software engineers who are responsible for building and maintaining our web and mobile applications, using cutting-edge technologies and agile methodologies.

### Job description
Join Withlocals as a **Senior Software Engineer** and create amazing web apps that connect travelers with local hosts! You will work in a fun and friendly team where you can share your ideas and learn from others. You will use **Next.js**, Typescript, React and more to build beautiful and user-friendly interfaces and services that showcase the best of each destination. You will also work closely with other engineers, designers and stakeholders to deliver features that delight our customers. You will follow best practices to ensure the quality and performance of our platform. You will also help us improve our codebase and architecture by applying your knowledge of modern web development trends and techniques.

### Profile
If you are looking for a challenging and rewarding career as a software engineer at Withlocals, you need to have the following skills. 

For hard skills, you need to be proficient or willing to learn at least some of these technologies: **Next.js**, **Typescript**, **MySQL**, **React**, **Vercel**, **Kubernetes**, **Google Cloud**. These technologies are essential for Withlocals for building fast, scalable and user-friendly web applications that connect travelers with local hosts. You should also be familiar with web development best practices such as testing, debugging and performance optimization.

For soft skills, you need to be **proactive** and **self-motivated**. You should be able to **work independently** and **take ownership** of your tasks. You should also be **business-minded** and **customer-focused**. You should understand the needs and expectations of both travelers and hosts and deliver solutions that meet or exceed them. 

You are curious and **eager to learn new technologies**. You need to keep yourself updated with the **latest trends** and **innovations** in web development and apply them to your projects. You should also be **pragmatic** and **results-oriented** and able to prioritize the most important outcomes that create value for Withlocals.

You are also an **excellent communicator** and **team player**. You should **collaborate** effectively with other engineers, designers, product managers and stakeholders within Withlocals. You should communicate clearly your ideas, progress and challenges using various tools such as Slack, Jira or Google Meet.

As a member of the Withlocals team, you are comfortable in a **hybrid work environment** that requires collaboration with both onsite and remote colleagues. You should be able to balance the big picture vision with the day-to-day goals, ensuring that your work is aligned with our overall strategy. And of course, you'll need to deliver **high-quality results** in a **timely manner**.

At Withlocals, we value **innovation** and encourage experimentation. We believe in the power of **trying new things** and learning from failures along the way. Our ideal candidate is passionate about crafting unforgettable experiences for travelers and shares our vision of empowering local hosts to showcase their culture through authentic activities.

Finally, you should be familiar with **product process** and best practices such as agile methodology, user research or prototyping. You should also be open to feedback and continuous improvement from your peers or customers.

### What you will get
If you join Withlocals as a software engineer, you will get more than just a job. You will get to work with awesome colleagues who are passionate about creating amazing experiences for travelers and locals alike. You will get **competitive compensation** that reflects your skills and contributions to our growing startup. You will get the latest MacBook Pro to help you code faster and better. And you will get flexible leave based on trust, so you can balance your work and life in the way that suits you best. Plus, you'll have the opportunity to grow your skills and knowledge with personal development time and budget for training.

### Our Commitment to Sustainability and Diversity
We love our planet and our people. That's why we at Withlocals are passionate about sustainability and diversity. We know that having a diverse team of talented and creative individuals makes us stronger and more innovative. We also know that being sustainable means being responsible for the impact we have on the environment and the communities we work with. Whether it's developing cutting-edge products, delivering exceptional services, or creating amazing solutions, we always strive to do it in a way that respects our planet and our people.

If you're up for the challenge, we'd love to have you join us. Get in touch with our CTO Bartosz at [bartosz@withlocals.com](mailto:bartosz@withlocals.com) to schedule an interview. We are excited to hear more about you!

**Comment by stevo2105** [comment]
[Whop](https://whop.com/) | Senior Full-Stack Engineer | In person (New York, NY) | Full-time | $170-240k base | [Link](https://careers.whop.com/26288)

Whop is a marketplace where people sell access to communities and software they create. We process over $10M / month and have almost a million users. Whop is hiring across the board. We are venture backed and looking for incredibly talented folks to join us in our NYC office. We are located in Brooklyn.

We have a small engineering team where you will be able to make incredible impact. Please reach out to me on [Twitter](https://twitter.com/cultured) or apply through the link above.

We are only hiring for in person roles at this time so please don't reach out unless you are willing to move to NYC. We have a massive opportunity on our hands and need to be completely aligned!

---

About Whop:

[Whop](https://whop.com/) is building a platform to service the internet economy. Millions of people are building new products that they are selling guerrilla style on social media and on forums throughout the internet. Whop gives these sellers a sleek storefront that can accept payments, seamlessly deliver digital products, and attract new customers visiting our marketplace. We currently handle over $100M+ in yearly payments with thousands of active merchants. To further our important mission of helping everyone start internet businesses, we are looking for an experienced senior engineer to help lead and grow a team. This role will oversee the major architectural changes, sprint planning, as well as aid in core infrastructure improvements and enhancements.

**About the role:**

Title: Senior Software Engineer

Salary: $170K - $240K | Equity: 0.1% - 0.3%

Location: Brooklyn, NY

Meals: Paid for lunch and dinner of your choice everyday

Gym: State of the art gym access in beautiful office building

Technology: Latest 16 inch Macbook Pro + 2 Studio Display Monitors

The role is broad in scope and requires the ability to switch contexts without losing sight of the most important objectives.

**Who you are**

You love high level architecture and debating implementations.

You are hard working, relentless, and understand that building something great takes grit.

You are exceptionally organized and efficient. You geek out over the latest technical implementations.

You love getting under the hood. You can dig into details, learn from mistakes, and figure out the underlying reasons for bad and good outcomes.

You love solving unsolved problems that are incredibly difficult.

If this sounds like you regardless of your background, we’d love to hear from you.

**Responsibilities**

Build and architect features from scratch.

Make our current codebase more scaleable.

Improve developer experience and onboarding for new developers joining our team.

Build scalable solutions on “how to” do things from a product standpoint.

**Tech**

Next.JS + Typescript

Tailwind

Vercel

Notion, Slack, GitHub

Linear

**Comment by mark-poulsen** [comment]
indyRIOT | Senior Full-Stack Engineer | Remote (Europe +/- 2 hours CET) | Full-time | $80-90k base | [Link](https://www.linkedin.com/jobs/view/3527051301/)

We’re on a mission to **create thriving online communities**, and want you to consider joining our engineering team!

Our stack leverages technologies like Next.js, tRPC, TypeScript and Serverless, so we can move fast, with high quality, while incorporating feedback to iterate our product.

We believe privacy is changing, that people deserve a better, more trusted online experience, and that through our platform, they can connect and share in better ways than current social media.


**That’s the pitch. Here’s what we’re looking for in our next engineers.**


- 6+ years experience as a full-stack engineer
- Deep knowledge, skills and experience with Typescript, React and ideally Next.js.
- Strong experience with deploying to cloud environments and optionally with Serverless technology like Vercel, database technology like PlanetScale and Prisma.
- Strong experience working within product development
- We value applications from Senior engineers, and by Senior, we mean that you take pride in enabling other people to do their best work, while delivering yours.


**In return, we’re offering**


- Full time employment, with the option to work from our Oslo offices, from home or remotely within Europe.
- A competitive salary with access to our options scheme.
- A senior position, meaning you have the chance to help lead and form our early technical decisions.
- A great company culture where we believe in kicking-ass and having fun!
- Great colleagues that care about our velocity, standards, well-being and the company’s mission
- An engineering manager who will help coach and support you - that's me, so feel free to contact me directly!

Sounds interesting? Contact me, use the link to apply, or reach us at careers@indyriot.com

See more information about IndyRIOT on our website: https://indyriot.com/

For any questions on the role or help with applying, please contact Mark Poulsen: mark.poulsen@indyriot.com

We have an inclusive environment with lots of tolerance! We see our differences as something essential to make a good product and will work actively to ensure candidates to this role reflect the variety of society; we welcome applications from every intersection of race, religion or belief, ethnic origin, different physical ability, family structure, socio-economics, age, nationality or citizenship, marital, domestic or civil partnership status, sexual orientation or gender identity.
